src/test/java/bubbles/ParserTest.java

```diff
@@ -0,0 +1,22 @@
+package bubbles;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void testFormatTask() {
```

Comment:
Maybe indicating the input in the method name here would get closer to the module's code requirements?